src/libs/actions/Policy/Policy.ts

```diff
@@ -5475,8 +5462,8 @@ function updateInterestedFeatures(features: Feature[], policyID: string) {
     }> = [];
 
     features.forEach((feature) => {
-        // If the feature is not enabled by default and it's programmatically enabled, we need to enable it
-        if (!feature.enabledByDefault && feature.programmaticallyEnabled) {
```

Comment:
I didnt have to change `programmaticallyEnabled` but i did cause personally i found it harder to read. Like i wasnt sure why its called "programmatically", when in reality it was enabled by the user. I just kept it as `enabled` so its easier to read.